### DISCUSSION TITLE
[@astrojs/image] Privacy & Security feature: Strip Image Metadata

### DISCUSSION THREAD

**Discussion by L1lith** [withastro/roadmap#249] [Proposal]
Stripping image metadata (like EXIF) would be a very nice privacy feature (that would be relatively difficult to implement yourself though possible). Images often include sensitive data about the user's location, etc. This could be toggled on and off as well theoretically. There could also be a plugin to extend this functionality to the `/public` directory too or maybe just all images in general?

**Comment by jasikpark** [comment]
This would still be a problem if you check the source images into git and the repo is available, the source images would need the EXIF data stripped

**Reply by L1lith** [reply]
What if it's not public though?

**Reply by jasikpark** [reply]
fair; would also be nice if it could warn you that you have location data embedded in your photos on dev/build

**Reply by jasikpark** [reply]
so you might catch it before you commit the files
